src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer_BuildSynchronization.cs

```diff
@@ -89,7 +89,7 @@ public async Task SynchronizeWithBuildAsync(ImmutableDictionary<ProjectId, Immut
                     }
 
                     // Enqueue re-analysis of open documents.
-                    AnalyzerService.Reanalyze(Workspace, documentIds: Workspace.GetOpenDocumentIds(), highPriority: true);
+                    AnalyzerService.Reanalyze(Workspace, documentIds: Workspace.GetOpenDocumentIds());
```

Comment:
❔ Why this change?

Reply:
Ah, let me revert - did not intend it here.